src/pybind/mgr/dashboard/tests/__init__.py

```diff
@@ -13,7 +16,16 @@ def __init__(self, retcode, message):
 def exec_dashboard_cmd(command_handler, cmd, **kwargs):
     cmd_dict = {'prefix': 'dashboard {}'.format(cmd)}
     cmd_dict.update(kwargs)
-    ret, out, err = command_handler(cmd_dict)
+    if cmd_dict['prefix'] not in CLICommand.COMMANDS:
+        ret, out, err = command_handler(cmd_dict)
+        if ret < 0:
+            raise CmdException(ret, err)
+        try:
+            return json.loads(out)
+        except ValueError:
```

Comment:
Python 3 raises a `JSONDecodeError`. Also in line 33.

Reply:
Since `JSONDecodeError` is a subclass of `ValueError` we are fully covered here, right?